typespec-extension/src/code-model-builder.ts

```diff
@@ -1707,7 +1706,7 @@ export class CodeModelBuilder {
         return this.processConstantSchemaForLiteral(type, nameHint);
 
       case "Enum":
-        return this.processChoiceSchema(type, this.getName(type), isFixed(this.program, type));
+        return this.processChoiceSchema(type, this.getName(type), true);
```

Comment:
@haolingdong-msft 

I think Chenjie said latest TCGC would have this same interpretation, that TypeSpec `enum` is fixed enum.